### PR TITLE
Initial contributing guide

### PR DESCRIPTION
**Pull request by avelino**

### CODE REVIEW

**Comment by avelino** [comment]
Team review text please

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+4. Please give all relevant information below for bug reports, incomplete details will be handled as an invalid report.
+
+**You MUST delete the content above including this line before posting, otherwise your issue will be invalid.**
+
+- pREST version (or commit ref):
+- PostgreSQL version:
+- Operating system:
+- Go version:
+- Log gist:
+
```
Comment (felipeweb) [inline .github/issue_template.md]: we can ask for a URL too, because pREST use querystring to build where SQL clause

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+If unique, [open an issue](https://github.com/nuveo/prest/issues/new) and answer the questions so we can understand and reproduce the problematic behavior.
+
+To show us that the issue you are having is in pREST itself, please write clear, concise instructions so we can reproduce the behavior (even if it seems obvious). The more detailed and specific you are, the faster we can fix the issue. Check out [How to Report Bugs Effectively](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html).
+
+Please be kind, remember that pREST comes at no cost to you, and you're getting free help.
+
+## Discuss your design
+
+The project welcomes submissions but please let everyone know what you're working on if you want to change or add something to the pREST repositorie.
```
Comment (felipeweb) [inline CONTRIBUTING.md]: fix typo to repository

**Comment by avelino** [comment]
@marioidival create new issue pls

**Pull request merged**